src/modules/AdvancedPaste/AdvancedPaste/Helpers/AICompletionsHelper.cs

```diff
@@ -127,6 +129,16 @@ public AICompletionsResponse AIFormatString(string inputInstructions, string inp
                 apiRequestStatus = -1;
             }
 
+            int promptTokens = -1;
+            int completionTokens = -1;
+            if (rawAIResponse != null)
```

Comment:
Should be `!string.IsNullOrEmpty(rawAIResponse)`?

Reply:
Fixed this by putting the telemetry event in the try block so we don't have to check for null